subprojects/model-core/src/main/java/org/gradle/util/internal/ConfigureUtil.java

```diff
@@ -32,6 +32,23 @@
 
 import static org.gradle.util.internal.CollectionUtils.toStringList;
 
+/**
+ * Contains utility methods to configure Groovy objects.
+ * <p>
+ * Plugins should avoid using this class, as the implementation will be bound to the Groovy. Instead, the object configuration should be  modeled with {@link Action}s, which is language-agnostic.
+ * Here's an example pseudocode:
+ * <pre>
```

Comment:
We should try to test this, WDYT? https://github.com/gradle/gradle/commit/211c5bd40fbe7f2ad6bc13f5791ebb38c31e29ff

Reply:
Looks good, the commit only needed a few cosmetic adjustments